src/resource/resource.test.ts

```diff
@@ -701,3 +708,68 @@ describe("cloneResource", () => {
     expect(clonedObject).not.toBe(sourceObject);
   });
 });
+
+describe("getContainedResourceUrlAll", () => {
+  const initContainer = (
+    containerUrl: string,
+    containedResourceUrls: UrlString[]
+  ) => {
+    let childrenIndex = createThing({ url: containerUrl });
+    childrenIndex = addUrl(childrenIndex, rdf.type, ldp.BasicContainer);
+
+    let mockContainer = mockContainerFrom(containerUrl);
+
+    containedResourceUrls.forEach((thingName) => {
+      let childListing = createThing({
+        url: containerUrl + thingName + ".ttl",
+      });
+      childListing = addUrl(childListing, rdf.type, ldp.Resource);
+      childListing = addDatetime(childListing, dct.modified, new Date());
+
+      childrenIndex = addUrl(childrenIndex, ldp.contains, childListing);
+      mockContainer = setThing(mockContainer, childListing);
+    });
+
+    mockContainer = setThing(mockContainer, childrenIndex);
+    childrenIndex = addUrl(childrenIndex, rdf.type, ldp.Container);
```

Comment:
Technically it doesn't matter since `getContainedResourceUrlAll()` doesn't check the `rdf:type` anyway, but this second line now doesn't do anything. (I'd also assume it to be next to the line in which you added the `BasicContainer` type, although both could also just be removed.)

```suggestion
    childrenIndex = addUrl(childrenIndex, rdf.type, ldp.Container);
    mockContainer = setThing(mockContainer, childrenIndex);
```